app-frontend/src/app/services/vendor/planetLabs.service.js

```diff
@@ -100,7 +100,9 @@ export default (app) => {
                         sunAzimuth: feature.properties.sun_azimuth,
                         sunElevation: feature.properties.sun_elevation
                     },
-                    statusFields: {}
+                    statusFields: {},
+                    // eslint-disable-next-line no-underscore-dangle
+                    planetPerm: feature._permissions
```

Comment:
just name it permissions - other repositories might have a similar restriction and use the same field in the frontend